comp/schedule.html

```diff
@@ -69,10 +69,9 @@ <h2>[[ session.description ]]</h2>
         <tr>
             <th data-ng-if="!session.show_arena_title">Time</th>
             <th data-ng-if="!session.show_arena_title">Match</th>
-            <th data-ng-repeat-start="arena in session.arenas">Corner 0</th>
-            <th>Corner 1</th>
-            <th>Corner 2</th>
-            <th data-ng-repeat-end>Corner 3</th>
+            <th data-ng-repeat="corner in session.all_corners track by $index">
```

Comment:
Just want to double check my understanding on this. We're using `all_corners` as we want to handle having multiple arenas. We don't need to update the body of this table as it already renders a column for each team in the match, and we only need to repeat over the teams in the match as a match can have multiple arenas so `match.teams` would contain the teams across all arenas?

Reply:
Yes.

Reply:
Thanks :)